### DISCUSSION TITLE
EasyPrivacy is blocking plausible.js

### DISCUSSION THREAD

**Discussion by ItalyPaleAle** [plausible/analytics#938] [Anything Else]
Looks like `/plausible.js` is now blocked in the EasyPrivacy list.

Is there a way to rename the JS script?

**Comment by ItalyPaleAle** [comment]
Update: looks like it's possible to proxy the script with a different URL.

Gotta love this cat-and-mouse game :)

**Reply by metmarkosaric** [reply]
The first solution is to serve the script from your domain as a first-party connection https://plausible.io/docs/custom-domain

And if you want even more, you can proxy the script https://github.com/plausible/analytics/discussions/610#discussioncomment-291102

**Reply by ItalyPaleAle** [reply]
Thanks! I am self-hosting (precisely because I knew sooner or later the hosted service would be blocked). The fact that scripts named `plausible.js` (no matter what domain they're served on) are blocked now came as a surprise to me.

I managed to change the name of the script via a proxy, and also serve it from the same domain. I have other ideas, but I'll comment on #610.
